Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealitySceneSystemProfile.asset

```diff
@@ -26,8 +26,8 @@ MonoBehaviour:
   lightingScenes:
   - Name: DefaultLightingScene
     Path: Assets/MixedRealityToolkit.SDK/StandardAssets/Scenes/DefaultLightingScene.unity
-    Included: 1
-    BuildIndex: 1
+    Included: 0
```

Comment:
This change seems unrelated

Reply:
correct, i'll put the current ones back. nice catch!